bundler.gemspec

```diff
@@ -32,4 +32,6 @@ Gem::Specification.new do |s|
   s.bindir        = "exe"
   s.executables   = %w(bundle bundler)
   s.require_paths = ["lib"]
+
+  s.post_install_message = "Bundler and RubyGems.org are free for anyone to use, but maintaining them costs more than $20,000 USD every month. If you make a profit using Ruby, help us cover those costs so that we can keep the gem ecosystem free for everyone: https://rubytogether.org/join"
```

Comment:
I feel the message is too forward and could use a little more tact. Something like "Bundler and RubyGems.org is maintained through funds raised by Rubytogether.org. Please help us keep the gem ecosystem alive for you and others to use by a one time donation or becoming a member, see https://rubytogether.org/join".
It maybe a bit long but for this particular message, it is far more important to have a clear message with the right tone.